api/utils.go

```diff
@@ -20,6 +21,19 @@ func httpError(w http.ResponseWriter, err string, status int) {
 	http.Error(w, err, status)
 }
 
+func sendJSONMessage(w io.Writer, id, status string) {
+	message := struct {
```

Comment:
I decided to NOT import `docker/docker/pkg/jsonmessage` because it would required vendoring 3 pkgs (including term) it seemed overkill